src/ruler.js

```diff
@@ -260,12 +260,7 @@ export function extendRuler() {
 		_computeDistance(gridSpaces) {
 			if (!this.isDragRuler) {
 				return super._computeDistance(gridSpaces);
-			}
-			if (!this.dragRulerEnableTerrainRuler) {
```

Comment:
Removing this entire branch will probably hurt distance calculations over difficult terrain. Can you verify that this is not the case?

Reply:
Unfortunately enhanced terrain level is not working on V11. See issues on  https://github.com/ironmonk88/enhanced-terrain-layer/issues

I created another PR without this particular questionable changes.

Note: I opened this PR based on the message in the https://github.com/manuelVo/foundryvtt-drag-ruler/issues/273 issue.
This can be closed